framework/include/multiapps/MultiApp.h

```diff
@@ -207,7 +207,7 @@ class MultiApp : public MooseObject,
    * Whether or not this MultiApp should be restored at the beginning of
    * each Picard iteration.
    */
-  virtual bool needsRestoration() { return true; }
+  virtual bool needsRestoration() { return !_no_restore; }
```

Comment:
Is this overridden anywhere anymore? If not, can remove the `virtual`

Reply:
done